devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -16,7 +16,7 @@
  *
  */
 metadata {
-	definition (name: "Z-Wave Metering Dimmer", namespace: "smartthings", author: "SmartThings") {
+	definition (name: "Z-Wave Metering Dimmer", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch") {
```

Comment:
I don't know what this device actually is.  Is it a light switch that you put in your wall? I was searching for examples of a metering dimmer but didn't really come up with anything.  Same with the Z-Wave metering switch DTH.  But there's a decent chance that I'm wrong.